about.html

```diff
@@ -0,0 +1,121 @@
+                                <p class="p-tag2">A joyful celebration believing in the value of openness and sharing,creating
+                                    a positive change with people from over 80 countries is taking place in October, in Rwanda </p>
+                            </div>
+                            <div class="time">
+                                <p>Please contact us per Email for any further questions about our virtual learning summit</p>
+                                <p>uwimbabazidorcas65@gmail.com</p>
+                            </div>
+                        </article>
+                    </div>
+                </div>
+            </section>
+    
+            <section class="main-section">
+                <div class="main-container">
+                    <div class="summit">
+                        <h2>Creative Annual summit 2023</h2>
+                        <hr class="line">
+                        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga, eveniet quis? Fugit, officiis vel ducimus fugiat quas sint ipsa repellendus eos consequatur dignissimos doloribus!</p>
+                    </div>
+                    <div class="summit-logo">
+                        <div><img class="my-logo2"src="img/photography-logo-vector.jpg" alt="photography picture"></div>
+                    </div>
+                </div>
+            </section>
+    
+            <section>
+                <div class="main-container">
+                    <div class="summit">
+                        <h3>See the past CC Global summits</h3>
+                        <hr class="line">
+                        <p>Take a look at the two CC Global Summits which took palce in Kigali and in Rubavu </p>
+                    </div>
+                    <div class="past-event">
+                        <div class="past-event1">
+                            <div class="past-year">
+                                <h4>2023</h4>
+                                <p>CC  Global summit 2023 in Kigali</p>
+                            </div>
+                        </div>
+    
+                        <div class="past-event2">
+                            <div class="past-year">
+                                <h4>2022</h4>
+                                <p>CC Global Summit 2022 in Rubavu</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+    
+            <section class="footer footer-about" id="footer">
+                <div class="main-container">
+                    <div id="footer-con">
+                        <div class="logo">
+                            <a href="index.html"><img class="my-logo4"src="img/creative-logo.png" alt="creative logo image"></a>
+                            <img  class="md-footer-img" src="img/creative-logo2.png" alt="creative logo image">
+                        </div>
```

Comment:
**OPTIONAL** - The sponsor section is missing in the mobile version of this page, though you have done well so far 👏, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
![awrhwrh](https://github.com/Dorcas126/final-capstone-project-1/assets/73607512/387a8d3d-d851-4adb-84e4-2f83be7dffc9)
Design template:
![awrgawrg](https://github.com/Dorcas126/final-capstone-project-1/assets/73607512/60962f59-fe01-4343-aaac-a4544c2bf7c8)